### PR TITLE
Fix learn more link to open another tab

### PR DESCRIPTION
**Pull request by Neeteshking21**
# Description
Fixes # 180

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] User Interface
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have ```npm run format``` before commiting
- [x] I replicated the UI with the Design 

# ScreenShots (if any)

### CODE REVIEW

**Comment by Neeteshking21** [comment]
bug_fix

**Comment by KamalSinghKhanna** [comment]
thanks for the PR @Neeteshking21
LGTM!

**Review by KamalSinghKhanna — APPROVED** [review]
lgtm1

**Comment by KamalSinghKhanna** [comment]
@kaiwalyakoparkar  can you please review this?

File: assets/css/style.css [hunk 1]
```diff
@@ -297,6 +297,7 @@ h6 {
 .checkbox {
 	opacity: 0;
 	position: absolute;
+    margin-left: 38px;
```
Comment (kaiwalyakoparkar) [inline assets/css/style.css:300]: Please indent the this line of code

File: assets/css/style.css [hunk 2]
```diff
@@ -307,7 +308,7 @@ h6 {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
-    margin-left: 16px;
+    margin: 2px 0 0 38px;
```
Comment (kaiwalyakoparkar) [inline assets/css/style.css:311]: Please indent the this line of code as well

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by Neeteshking21 — COMMENTED** [review]
All fixed.

**Comment by Neeteshking21** [comment]
@kaiwalyakoparkar

**Pull request closed without merging**